RecoTracker/FinalTrackSelectors/plugins/TrackCutClassifier.cc

```diff
@@ -180,7 +180,7 @@ namespace {
     
     float operator()(reco::Track const & trk,
 		     reco::BeamSpot const & beamSpot,
-		     reco::VertexCollection const & vertices) const {
+		     reco::VertexCollection const & vertices) {
```

Comment:
what was the reason to make this non-const?

Reply:
You're right, there is no good reason (you would see that this came from me if the history were not squashed to a single commit).